ReleaseProcessAutomation/Steps/PipelineSteps/PushNewReleaseBranch.cs

```diff
@@ -0,0 +1,31 @@
+using System.Runtime.InteropServices.ComTypes;
+using ReleaseProcessAutomation.Configuration.Data;
+using ReleaseProcessAutomation.Git;
+
+namespace ReleaseProcessAutomation.Steps.PipelineSteps;
+
+public interface IPushNewReleaseBranch
+{
+  void Execute (string releaseBranchName, string firstAncestorBranchName);
+}
+
+public class PushNewReleaseBranch
```

Comment:
This class and its interface should have a `Step` suffix, otherwise they are located in the wrong folder